app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/CameraUtils.kt

```diff
@@ -0,0 +1,209 @@
+package com.example.android.architecture.blueprints.todoapp.data
+
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.*
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import com.example.android.architecture.blueprints.todoapp.R
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.FileOutputStream
+import java.io.InputStream
+import java.util.*
+
+fun getCircleBitmap(color: Int, size: Int, activityContext: Context): Drawable {
+    val a = Bitmap.createBitmap(size, size, Bitmap.Config.ARGB_8888)
+    val canvas = Canvas(a)
+    val paint = Paint()
+    paint.isAntiAlias = true
```

Comment:
here we can use kotlin apply scope function in paint such as paint.apply{
color=
style=..
}